main.css

```diff
@@ -94,15 +94,56 @@ footer
     background-color: #061428;
 }
 
-#sidebar
+#mobileMenuIcon
 {
     position: fixed;
+    top: 35px;
+    left: 5px;
+    width: 48px;
+    height: 48px;
+    z-index: 9999;
+    background: url("assets/mobile-ui/mobilemenu.png");
+}
+
+#mobileHideMenu
```

Comment:
This never seems to show up for me? Either way, we should have arrows instead, and probably use the same element for both maybe a `#toggleExpandContact` div that can be given/taken away the `#expandIcon`/`#contractIcon` id